gridcapa-core-cc-post-processing-app/src/test/java/com/farao_community/farao/core_cc_post_processing/app/PostProcessingServiceTest.java

```diff
@@ -35,8 +35,7 @@
 import static com.farao_community.farao.core_cc_post_processing.app.Utils.RAO_RESULT_FILE_DTO;
 import static com.farao_community.farao.core_cc_post_processing.app.Utils.SUCCESS_TASK;
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.*;
```

Comment:
It's better to import packages one by one

Reply:
sorry after update of intelli J I lost my configuration

Reply:
done